<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-RDP6H5E" name="Default test" tests="5" failures="1" timestamp="12 Mar 2021 01:19:18 GMT" time="6.404" errors="0">
  <testcase name="maintest" time="0.369" classname="signuptestscript.Scriptflickr"/>
  <testcase name="signupisdisplayed" time="0.072" classname="signuptestscript.Scriptflickr"/>
  <testcase name="validatetitle" time="0.022" classname="signuptestscript.Scriptflickr"/>
  <testcase name="validateurl" time="0.015" classname="signuptestscript.Scriptflickr"/>
  <testcase name="validateTermsURL" time="0.205" classname="signuptestscript.Scriptflickr">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;https://[www.flickr.com/help/terms]&amp;gt; but was:&amp;lt;https://[identity.flickr.com/sign-up]&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<https://[www.flickr.com/help/terms]> but was:<https://[identity.flickr.com/sign-up]>
	at junit.framework.Assert.assertEquals(Assert.java:100)
	at junit.framework.Assert.assertEquals(Assert.java:107)
	at signuptestscript.Scriptflickr.validateTermsURL(Scriptflickr.java:134)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validateTermsURL -->
</testsuite> <!-- Default test -->
